### PR TITLE
[citest skip] README: Clarify the name setting

### PR DESCRIPTION
**Pull request by tyll**
Signed-off-by: Till Maas <opensource@till.name>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,24 @@ For NetworkManager, a connection can only be active at one device each time.
...
 
+**Note:** The network role will only change the profiles that are specified in the
+`network_connections` variable. Therefore, if only the ports of a profile are specified
+to be removed from the controller and the controller is not specified, then the
+controller profile will remain on the system. This can happen, if for example all ports
+are removed from a bond interface.
+
+**Note:** To remove all profiles on a system that are not specified in the
+`network_connections` variable, add an entry without a name and `persistent_state:
+absent`. This will match and remove all remaining profiles:
```
Comment (liangwen12year) [inline README.md:133]: Does inserting a newline between different connections look better ? 

```yaml
network_connections:
  - name: eth0  # profiles to keep/configure on the system
    [...]

  - persistent_state: absent  # remove all other profiles
```

File: README.md [hunk 2]
```diff
@@ -122,6 +122,24 @@ For NetworkManager, a connection can only be active at one device each time.
...
+
+```yaml
+network_connections:
+  - name: eth0  # profiles to keep/configure on the system
+    [...]
+  - persistent_state: absent  # remove all other profiles
+```
+
+
```
Comment (liangwen12year) [inline README.md:142]: mdl error here.  (Multiple consecutive blank lines)

File: README.md (hunk 1 above)
Comment (tyll) [reply README.md:133]: Thanks, good catch, fixed it.

File: README.md (hunk 2 above)
Comment (tyll) [reply README.md]: thx, fixed.

**Review by liangwen12year — COMMENTED** [review]
Looks good to me, but rebasing is needed.

**Comment by tyll** [comment]
> Looks good to me, but rebasing is needed.

Could you please resolve the conversations and approve the PR if you agree. It seems to me that github can do the rebase itself afterwards.

**Review by liangwen12year — APPROVED** [review]

**Pull request merged**